docs/adr/0003-End-to-end-tests.md

```diff
@@ -0,0 +1,57 @@
+# 3. End to end tests
+
+Date: 2023-04-18
+
+## Status
+
+Accepted
+
+## Context
+
+End to end (e2e) tests only run tests against exporters `/metrics` endpoint in OpenShift today (using Prow CI). We want to run them with more cases, like against Grafana, making them more reliable.
+
+## Decision
+
+e2e tests will be more user friendly.
+
+Prow CI will run e2e tests that cover more scenarios and are more reliable.
+
+### How
+
+- We will rewrite the e2e tests script in Python.
+    - easier to write and read it
+    - they will run against Pelorus operator (using test image) and not helm
+
+- We will write better documentation on what e2e tests do (script help and read the docs).
+    - more accessible to new developers of the Pelorus project
+
+- We will log only relevant information on tests run.
+    - Too much log information is not good to identify a simple error
```

Comment:
Yes, but too little is also sometimes wrong, so we could have two logs. Summary and full log?

Reply:
we can have log levels, like in the exporters, and the default not being verbose (debug)

Reply:
or multiple log streams with different log level. I rather do not produce different format of e2e tests depending on some switch. If test breaks many times we don't have opportunity to check why afterwords, so all info necessary should be associated with every run. It's not user application, but development tool.